File: past_archive/boj5/4796.cc
//
// 캠핑
// Link: https://www.acmicpc.net/problem/4796
//

#include <stdio.h>

int Min(int x, int y) { return x < y ? x : y; }

int main() {
  int L, P, V;
  scanf("%d%d%d", &L, &P, &V);
  int case_count = 1;
  while (L + P + V != 0) {
    int ans = (V / P) * L + Min(V % P, L);

    printf("Case %d: %d\n", case_count++, ans);
    scanf("%d%d%d", &L, &P, &V);
  }
  return 0;
}

File: 24boj_py/4796.py
# 캠핑
# https://www.acmicpc.net/problem/4796

import sys

x = 1
while True:
    L, P, V = map(int, sys.stdin.readline().split())
    if L == 0 and P == 0 and V == 0:
        break
    sys.stdout.write(
        f'Case {x}: {L * (V // P) + min(L, V % P)}\n')
    x += 1
